src/pages/store/index.tsx

```diff
@@ -1,9 +1,45 @@
+import { ItemCard, Navbar } from '@/components/store';
 import withAccessType from '@/lib/hoc/withAccessType';
 import { PermissionService } from '@/lib/services';
+import { PrivateProfile } from '@/lib/types/apiResponses';
 import { GetServerSideProps } from 'next';
 
-const StoreHomePage = () => {
-  return <h1>Store Home Page</h1>;
+interface HomePageProps {
+  user: PrivateProfile;
+}
+const StoreHomePage = ({ user: { credits } }: HomePageProps) => {
+  // TEMP, just to demo the components
+  return (
+    <>
+      <Navbar balance={credits} />
+      <div
+        style={{
+          display: 'grid',
+          gap: '10px',
+          gridTemplateColumns: 'repeat(auto-fill, minmax(300px, 1fr))',
+          marginTop: '32px',
+        }}
+      >
+        <ItemCard
+          image="https://acmucsd.s3.us-west-1.amazonaws.com/portal/profiles/07789a16-8326-4edc-ad2d-fc6193cd1ee3.jpg"
+          cost={42069}
+          title="Nishant (not for sale)"
+          href="/leaderboard"
```

Comment:
```suggestion
          href="/store/items/uuid"
```
Can you mock more realistic data even if it doesn't exist and is the same for all items